plugins/slack_webhooks/app/models/slack_webhook_notification.rb

```diff
@@ -1,14 +1,17 @@
 class SlackWebhookNotification
-  def initialize(deploy)
+  def initialize(deploy, deploy_phase=:for_buddy)
     @deploy = deploy
+    @deploy_phase = deploy_phase
     @stage = deploy.stage
     @project = @stage.project
     @user = @deploy.user
   end
 
-  def deliver
+  def deliver(message=content)
```

Comment:
ok. I gave it a second try, separating the `deliver` and `buddy_request` methods. Should be better now.